File: sdk/go/microsandbox.go
// Package microsandbox is a minimal SDK for the Microsandbox project.
package microsandbox

import "fmt"

// Greet returns a greeting message for the given name.
func Greet(name string) string {
	message := fmt.Sprintf("Hello, %s! Welcome to Microsandbox!", name)
	fmt.Println(message)
	return message
}

File: sdk/dart/lib/microsandbox.dart
/// A minimal SDK for the Microsandbox project.
library microsandbox;

/// Returns a greeting message for the given name.
///
/// Example:
/// ```dart
/// var message = greet('World');
/// print(message); // Prints: Hello, World! Welcome to Microsandbox!
/// ```
String greet(String name) {
  final message = 'Hello, $name! Welcome to Microsandbox!';
  print(message);
  return message;
}
